x/emissions/keeper/inference_synthesis/synth_palette_weight.go

```diff
@@ -69,7 +69,7 @@ func (p *SynthPalette) CalcWeightsGivenWorkers() (RegretInformedWeights, error)
 				p.Logger.Debug(fmt.Sprintf("Cannot find worker in ForecasterRegrets in CalcWeightsGivenWorkers %v", worker))
 				continue
 			}
-			regretFrac, err := regretInfo.regret.Quo(stdDevRegretsPlusEpsilon)
+			regretFrac, err := regretInfo.Quo(stdDevRegretsPlusEpsilon)
```

Comment:
regretInfo name may be misleading now that there is no "info", should be regret instead?

Reply:
it includes the `noPriorRegret` bool

Reply:
It use too.. now it can be just regret indeed